tests/test_strategy_pvpredefined.py

```diff
@@ -19,7 +19,7 @@
 import pendulum
 import uuid
 import pathlib
-from pendulum import DateTime, duration, today
+from pendulum import DateTime, duration, today  # NOQA
```

Comment:
Why is the NOQA useful here?

Reply:
Removing, since one of the test was commented for a while. Had to do this then to avoid flake8